commitchecker.yaml

```diff
@@ -0,0 +1,4 @@
+upstreamOrg: operator-framework
+upstreamRepo: rukpak
+upstreamBranch: main
+expectedMergeBase: 0e327c385ca5488da4c2eca95a68d385c7e08a5f
```

Comment:
What is this line for? Seems a bit too specific?

Reply:
We are always very explicit about which upstream commit we're based off of. This ensures that we're correct.

Reply:
1. Does this change frequently?
2. As far as I can tell, this is the oldest commit where `UPSTREAM` use started?

Reply:
It will change every time the bot changes it, so perhaps daily.